packages/markdown-render-engine/README.md

```diff
@@ -0,0 +1,126 @@
+# Markdown render engine
+
+## Overview
+
+This render engine is based on [react-markdown](https://github.com/rexxars/react-markdown) component.
+
+## Installation
+
+```bash
+npm i @kyma-project/dc-markdown-render-engine
+```
+
+## Options
+
+> **NOTE**: This render engine support all options available in [react-markdown](https://github.com/rexxars/react-markdown#options) component.
+
```

Comment:
Options from `https://github.com/rexxars/react-markdown`? I think link will be better, because package has a lot of configs and options and duplicating content from original Readme is unnecessary.